test/Microsoft.ML.Functional.Tests/IntrospectiveTraining.cs

```diff
@@ -182,15 +182,24 @@ public void InspectLdaModelParameters()
 
             // Define the pipeline.
             var pipeline = mlContext.Transforms.Text.ProduceWordBags("SentimentBag", "SentimentText")
-                .Append(mlContext.Transforms.Text.LatentDirichletAllocation("Features", "SentimentBag", numberOfTopics: numTopics, maximumNumberOfIterations: 10));
+                .Append(mlContext.Transforms.Text.LatentDirichletAllocation("Features", "SentimentBag", 
+                numberOfTopics: numTopics, maximumNumberOfIterations: 10));
```

Comment:
_(appended to code to allow for threaded discussion)_
Should we also add the `outputWordSummary` option to the API found here?:

https://github.com/frank-dong-ms/machinelearning/blob/aa75df98f549741893deb3a042a5dd287a56ab99/src/Microsoft.ML.Transforms/Text/TextCatalog.cs#L597-L614

Reply:
That will be break public API, I think we should avoid break public API as much as possible. User still can get summary via the public interface from lda transformer

---
In reply to: [448136744](https://github.com/dotnet/machinelearning/pull/5260#discussion_r448136744) [](ancestors = 448136744)

Reply:
Or I can try to add a new API with same name but one more parameter as outputWordSummary

---
In reply to: [448512997](https://github.com/dotnet/machinelearning/pull/5260#discussion_r448512997) [](ancestors = 448512997,448136744)

Reply:
You're right. I don't think it's a good idea to add this option to this public API. No need for a new API only for this.

---
In reply to: [448522335](https://github.com/dotnet/machinelearning/pull/5260#discussion_r448522335) [](ancestors = 448522335,448512997,448136744)